model/riscv_sys_regs.sail

```diff
@@ -403,10 +403,13 @@ function legalize_tvec(o : Mtvec, v : xlenbits) -> Mtvec = {
  match (trapVectorMode_of_bits(v[Mode])) {
    TV_Direct => v,
    TV_Vector => v,
+   TV_Smclic => v,
    _         => [v with Mode = o[Mode]]
  }
 }
 
+function haveSmclic()  -> bool = mtvec[Mode] == 0b11
```

Comment:
haveFoo means Foo is available for use, not that its additions are currently being used. That is, haveSmclic means "is the model configured to provide the Smclic extension?", not "has software configured mtvec to use the CLIC?".

Reply:
I can change all places that say haveSmclic to SmclicMode.  Is that acceptable?